main.go

```diff
@@ -197,8 +197,15 @@ func main() {
 
 	runner := NewDumpRunner(basePath)
 
-	log.Print("Running dump tasks...")
+	log.Print("Running dump and analyse tasks...")
```

Comment:
hmm, I'm not sure... I think you analyse something to get an analysis as the result? So analyse is the action and analysis is the object? 

I'm getting semantic satiation on this word now!